provider/aws/formation/rack.json

```diff
@@ -68,29 +127,29 @@
-      "ap-southeast-1": { "Ami": "ami-0f9e7ad4abf49df6b", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "114774131450", "Fargate": "Yes" },
-      "ap-southeast-2": { "Ami": "ami-0cdf16bb10adcaaa1", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "783225319266", "Fargate": "Yes" },
-      "ca-central-1":   { "Ami": "ami-0a5cfc26b04c4a20e", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "985666609251", "Fargate": "No"  },
-      "eu-central-1":   { "Ami": "ami-0da383d2d6bdb9100", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "054676820928", "Fargate": "Yes" },
-      "eu-north-1":     { "Ami": "ami-02ed85945b0e46f6f", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "897822967062", "Fargate": "Yes" },
-      "eu-south-1":     { "Ami": "ami-00d7a4084bac16c81", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "635631232127", "Fargate": "Yes" },
-      "eu-west-1":      { "Ami": "ami-0fdb1527ad8bf80f7", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "156460612806", "Fargate": "Yes" },
-      "eu-west-2":      { "Ami": "ami-0b5e334d61108a1aa", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "652711504416", "Fargate": "Yes" },
-      "eu-west-3":      { "Ami": "ami-004138ffbf7aad984", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "009996457667", "Fargate": "Yes" },
-      "me-south-1":     { "Ami": "ami-05f7a182e037b82af", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "076674570225", "Fargate": "Yes" },
-      "sa-east-1":      { "Ami": "ami-00f653b3399483762", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "507241528517", "Fargate": "Yes" },
-      "us-east-1":      { "Ami": "ami-04942e54b391af5d0", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "127311923021", "Fargate": "Yes" },
-      "us-east-2":      { "Ami": "ami-0c38a2329ed4dae9a", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "033677994240", "Fargate": "Yes" },
-      "us-west-1":      { "Ami": "ami-0ab1b97c4bbf16bf3", "EFS": "Yes", "ThirdAvailabilityZone": "No",  "ELBAccountId": "027434742980", "Fargate": "No"  },
-      "us-west-2":      { "Ami": "ami-06c73a8827b372409", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "797873946194", "Fargate": "Yes" },
-      "us-gov-east-1":  { "Ami": "ami-04f633b669ee7b61c", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "190560391635", "Fargate": "Yes" },
-      "us-gov-west-1":  { "Ami": "ami-90704ef1",          "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "048591011584", "Fargate": "Yes" }
+      "af-south-1":     { "Ami": "ami-0e71c51acff69c762", "ArmAmi": "",                      "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "098369216593", "Fargate": "Yes" },
```

Comment:
No results for this region, nor `ap-northeast-3`?

Reply:
Yeah, "af-south-1" doesn't even appear on the ARM list here https://docs.aws.amazon.com/pt_br/AmazonECS/latest/developerguide/ecs-optimized_AMI.html

"ap-northeast-3" has an entry but following the link just redirects to the main System Manager UI.
I get nothing via the cli too:
```
aws ssm get-parameter --name "/aws/service/ecs/optimized-ami/amazon-linux-2/arm64/recommended/image_id" --region ap-northeast-3 
An error occurred (ParameterNotFound) when calling the GetParameter operation: 
```

Reply:
Okay, yeah, let's not worry about that then!